src/cdk/overlay/overlay.md

```diff
@@ -7,7 +7,7 @@ using the CDK on its own, you'll have to include the styles yourself. You can do
 the prebuilt styles in your global stylesheet:
 
 ```scss
-@import '~@angular/cdk/overlay-prebuilt.css';
+@use '~@angular/cdk/overlay-prebuilt.css';
```

Comment:
This one was actually a native CSS `@import` rather than a Sass import (because of the `.css` extension).

Reply:
Reverted, although I think that the result will be the same for Webpack projects where `@import` statements inline their result.